tesseract_python/tesseract_python/swig/tesseract_planning/tesseract_motion_planners/trajopt/trajopt_motion_planner.i

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file trajopt_motion_planner.i
+ * @brief SWIG interface file for tesseract_planning/trajopt/config/trajopt_motion_planner.h
+ *
+ * @author John Wason
+ * @date December 10, 2019
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2019, Wason Technology, LLC
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+%{
+#include <tesseract_motion_planners/trajopt/trajopt_motion_planner.h>
+%}
+
+%shared_ptr(tesseract_motion_planners::TrajOptMotionPlannerStatusCategory)
+
+namespace tesseract_motion_planners
+{
+class TrajOptMotionPlannerStatusCategory;
+
+class TrajOptMotionPlanner : public MotionPlanner
+{
+public:
+  
+  TrajOptMotionPlanner(std::string name = "TRAJOPT");
+
+  ~TrajOptMotionPlanner() {}
+
+  //bool setConfiguration(const TrajOptPlannerConfig::Ptr config);
+  %extend {
```

Comment:
Why the change?

Reply:
To match the Python convention of throwing exceptions on failure.